node/src/main/scala/io/casperlabs/node/configuration/ChainSpec.scala

```diff
@@ -50,17 +50,25 @@ object ChainSpec extends ParserImplicits {
       patch: Int
   )
 
+  final case class DeployConfig(
+      minTtlMillis: Int Refined NonNegative,
+      maxTtlMillis: Int Refined NonNegative,
+      maxDependenciesNum: Int Refined NonNegative
```

Comment:
```suggestion
      maxDependencies: Int Refined NonNegative
```
I wouldn't use this `Num` postfix, it's pretty self explanatory and half the variables are just `maxDependencies` anyway. Reminds of the `validator_block_seq_num` which is an index.

Reply:
```suggestion
  ) extends SubConfig
```
That's what the Magnolia generators look for.